src/nvim/api/extmark.c

```diff
@@ -366,9 +366,6 @@ Array nvim_buf_get_extmarks(Buffer buffer, Integer ns_id, Object start, Object e
 
 /// Creates or updates an extmark.
 ///
-/// To create a new extmark, pass id=0. The extmark id will be returned.
```

Comment:
This is essential information to use the API and should be corrected, not removed. Something like "By default a new extmark is created. To move an existing mark, pass its id."